better css class names

jantimon opened the pull request:
![shot-sIr9gJpE](https://github.com/jantimon/next-yak/assets/4113649/3792ca35-13c5-40a5-8b95-405078e0ddaf)
![shot-udXCnq3g](https://github.com/jantimon/next-yak/assets/4113649/9ef99027-c3cd-4d52-80d2-d7ab4c1c2ac9)

Mad-Kat commented on packages/next-yak/loaders/lib/localIdent.cjs:
```diff
@@ -17,15 +17,23 @@
+ * @param {number} i
+ */
+function unreadableNumber(i) {
+  return i.toString(25).replace(/\d/g, (m) => (Number(m)+26).toString(36));
```
I don't think we need such a "complicated" logic for this problem. At the moment it generates strings beginning from "q" instead of "0" and iterates through the alphabet. Can we just start with "a" if numbers are to distracting? Or keep it simple and a number for the moment 😄

jantimon replied:
it's like a hex algorithm but without numbers - but yes probably to complicated 😄

0 'q'
VM330:2 1 'r'
VM330:2 2 's'
VM330:2 3 't'
VM330:2 4 'u'
VM330:2 5 'v'
VM330:2 6 'w'
VM330:2 7 'x'
VM330:2 8 'y'
VM330:2 9 'z'
VM330:2 10 'a'
VM330:2 11 'b'
VM330:2 12 'c'
VM330:2 13 'd'
VM330:2 14 'e'
VM330:2 15 'f'
VM330:2 16 'g'
VM330:2 17 'h'
VM330:2 18 'i'
VM330:2 19 'j'
VM330:2 20 'k'
VM330:2 21 'l'
VM330:2 22 'm'
VM330:2 23 'n'
VM330:2 24 'o'
VM330:2 25 'rq'
VM330:2 26 'rr'
VM330:2 27 'rs'
VM330:2 28 'rt'
VM330:2 29 'ru'
VM330:2 30 'rv'
VM330:2 31 'rw'
VM330:2 32 'rx'
VM330:2 33 'ry'
VM330:2 34 'rz'
VM330:2 35 'ra'
VM330:2 36 'rb'
VM330:2 37 'rc'
VM330:2 38 'rd'
VM330:2 39 're'
VM330:2 40 'rf'
VM330:2 41 'rg'
VM330:2 42 'rh'
VM330:2 43 'ri'
VM330:2 44 'rj'
VM330:2 45 'rk'
VM330:2 46 'rl'
VM330:2 47 'rm'
VM330:2 48 'rn'
VM330:2 49 'ro'

jantimon commented:
I guess instead of making the number more readable we should try to remove the number whenever the class name exists only once in the file.

So I removed `unreadableNumber` again.

I also added tests

Mad-Kat commented:
Strongly agree with your take and thank you for the additional tests :)
